common/src/services/import.service.ts

```diff
@@ -209,7 +208,7 @@ export class ImportService implements ImportServiceAbstraction {
   }
 
   getImporter(
-    format: ImportType,
+    format: ImportType | "bitwardenpasswordprotected",
```

Comment:
do we still need this format? My understanding is that we can just import with `bitwardenjson` and it will figure it out.

Also applies to line 223.

I think I also saw it in the cli PR, maybe just do a global search (assuming that my understanding is correct).

Reply:
There are a couple of points here that as I write them sounds like excuses, but here they are:

The `ImportType` type is used to display the known import types with clients. Since `bitwardenpasswordprotected` is meant to be a transparent, internal alternative to `bitwardenjson`, we don't want to include it in that display. Since there's only one, I felt the best course right now is to just tack it on here (the only place it's needed) rather than try and figure out a good permanent home for these internal-only importers. I really can't see the use for another one atm.

We can get rid of the importer class entirely if we want to roll all the password protected. However, I think that the way to do that properly is to add a `parse_with_password` method for specifically this importer. It just feels weird since the way we currently alter importer behavior is with the importer type. It also feels weird to expand the `bitwardenjson` importer to include another json variant.

Reply:
Good explanation thank you, I agree that it's a little awkward being tacked on but it seems OK given that it's the only internal-only importer.